README.asciidoc

```diff
@@ -1375,6 +1375,32 @@ https://github.com/elastic/docs/blob/master/conf.yaml[`conf.yaml`].
 The path should be as specific as possible because we skip rebuilding books if
 changes to the referenced repository don't change the referenced path.
 
+[[glossary-pop-ups]]
+== Inline pop-ups for glossary terms
+
+Rather than linking to glossary pages, you can use an inline definition pop-up.
+
+[source,asciidoc]
+----
+definition:["word", "definition"]
+----
+
+Definition pop-ups can be assigned to attributes, consume attributes, and contain links.
+To use special characters in definition pop-ups, including commas, wrap the definition in double quotation marks.
+
+[source,asciidoc]
+----
+// Assign the definition to an attribute
+:run-short-def: To move at a speed faster than a walk. See <<run-link,run>> to learn more.
+
+// Create a definition pop-up and assign it to an attribute
+:run-def: definition:[run,"{run-short-def}"]
+
+// Reference the definition attribute to create a pop-up in the docs
+run::
+{run-short-def}. I like to {run-def}.
+----
```

Comment:
Maybe also include this as actual Asciidoc (outside the `[source]` tag) so people previewing the README can see it.